sct-commons/pom.xml

```diff
@@ -99,12 +99,6 @@
             <artifactId>jakarta.annotation-api</artifactId>
```

Comment:
This dependency is never used.  It is unaffected by the upgrade, so we can remove it.
> [!WARNING]
> [INFO] --- maven-dependency-plugin:3.3.0:analyze (default-cli) @ sct-commons ---
[WARNING] Used undeclared dependencies found:
......
[WARNING]    jakarta.annotation:jakarta.annotation-api:jar:1.3.5:compile
...

Reply:
ok